assets/i18n/en.json

```diff
@@ -603,13 +603,19 @@
     "I18N_FOOTER_BROWSE_LIBRARY": "Browse the Library",
     "I18N_FOOTER_CONTACT_US": "Contact Us",
     "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "CONTRIBUTE",
+    "I18N_FOOTER_COPYRIGHT": "© <[currentYear]> The Oppia Foundation",
     "I18N_FOOTER_CREDITS": "Credits",
     "I18N_FOOTER_DONATE": "Donate",
     "I18N_FOOTER_FOLLOW_US": "Follow Us",
     "I18N_FOOTER_FORUM": "Forum",
     "I18N_FOOTER_GET_INVOLVED": "Get Involved",
     "I18N_FOOTER_GET_STARTED": "Get Started",
+    "I18N_FOOTER_NEWSLETTER_EMAIL": "Email Address",
+    "I18N_FOOTER_NEWSLETTER_HEADING": "Subscribe To Our Newsletter",
+    "I18N_FOOTER_NEWSLETTER_TEXT": "Be a part of the Oppia community! Our bi-weekly newsletter keeps you informed on new features, lessons and how Oppia is making a worldwide impact.",
     "I18N_FOOTER_OPPIA_FOUNDATION": "The Oppia Foundation",
+    "I18N_FOOTER_OPPIA_MISSION": "Oppia’s mission is to provide a learning platform that meets the unique needs for under resourced communities.",
+    "I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER": "<a href=\"/donate\">Donate</a> or <a href=\"/volunteer\">Volunteer</a> today!",
```

Comment:
Does this have to be an HTML snippet?

Reply:
Maybe not necessarily, but since we're altering the color of the `Donate` and `Volunteer` anchor tags, not doing so will add some extra logic because we're using i18n instead of writing it in the HTML file itself.